### ISSUE TITLE
An unhandled exception of type 'System.TypeInitializationException' occurred in Emgu.CV.dll

### ISSUE THREAD

**Issue by Yotoh** [AlbinoDrought/LoveBoot#21]
// load all images based on Signal types. throws error if any of these are not found
            foreach (Signal s in Enum.GetValues(typeof(Signal)))
            {
                string subImagePath = String.Format("{0}{1}{2}", IMAGE_PATH, s.ToString(), IMAGE_EXT);
                imageFinder.SubImages.Add(s, new Image<Bgr, byte>(subImagePath));
            }

An unhandled exception of type 'System.TypeInitializationException' occurred in Emgu.CV.dll

Additional information: The type initializer for 'Emgu.CV.CvInvoke' threw an exception.

I haven't messed with the path can you help

**Comment by fkl233** [comment]
Yotoh I have a problem with images recognition for boot in korean version, don't detect touch, Do you have any idea how to solve this problem?

**Comment by Yotoh** [comment]
No I don't I am still trying to solve this issue myself if you have any answer it'd be very much appreciated.

**Comment by AlbinoDrought** [comment]
Did you extract the archive fully (including the Emgu dlls)?

**Comment by Yotoh** [comment]
Yes I have completely extracted it everything is there but I still will get the same errors

**Comment by Marieth63** [comment]
AlbinoDrought I have the images when space bar get 100%  what  should I do?  And  help me for korean version :( I can't enter latin version, I have accounts for lovebeat kr.

**Comment by dereckje** [comment]
South Korea currently has two automatic programs flourished. And receive Instant Confirmation stop distributing the source code for the program stops and program distribution. Thank you as Korea LoveBeat users.

**Comment by dereckje** [comment]
In addition, the buying and selling of money in the program is in progress. It plans to punish the person who sold the program to step on the current legal proceedings.

**Comment by Marieth63** [comment]
Are you Korean ? Do you know about the programs? 
I think AlbinoDrought sold the program to someone that doesn't want to share for the Korean version

**Comment by dereckje** [comment]
I don't think so. This program use bypass to Check Program. AlbinoDrought !! Please delete all the downloaded data available. Korea LoveBeat because you can die.

**Comment by AlbinoDrought** [comment]
This program has no bypasses at all, and does not even really read the
games memory. It also tries to get you banned by saying you're use
LoveBoot....

**Comment by Marieth63** [comment]
Hello AlbinoDorught that is a lie because in lovebeat korean exist many boots for doo all combo. You will not have any problem with this because it is just a game and you are not doing anything wrong. The only he says that because he doesn't want to share your boot with other users. I'm a user of lovebeat Korea'm waiting a long time that you can upload the boot for version Korea. I would be grateful ago.

**Comment by fkl233** [comment]
AlbinoDorught I'm also waiting for many months for the boot  korean version, please if you don't want to upload could you share privately? I wait your answer. You're great.

**Comment by dereckje** [comment]
I am very sorry according to the developer. South Korea LoveBeat user due to the sharing of those files are wearing the damage. :(

**Comment by dereckje** [comment]
게임을 우회하여 프로그램은 한국 LoveBeat에서 작동한다. 메모리값을 읽지 못한다고? 현재 프로그램을 이용해 게임에서 불법적으로 경험치를 얻고있다. 개발자에게 부탁한다 한국의 LoveBeat를 위해 공유를 중단해라.

**Comment by dereckje** [comment]
Bypassing the game program will operate in Korea LoveBeat. You can not read the memory values? Gaining experience in the game unlawfully using the current program. Ask the developers Be stop sharing for LoveBeat of Korea.

**Comment by dereckje** [comment]
In addition the program can not open your popup window by pressing twice F10. Is it possible accusations that the program is properly made? The source code I hope you did not share. Please, please stop sharing. please.
